.bcr/source.template.json

```diff
@@ -1,5 +1,5 @@
 {
   "integrity": "",
-  "strip_prefix": "rules_platform-0.0.0",
-  "url": "https://github.com/bazelbuild/rules_platform/releases/download/0.0.0/rules_platform-0.0.0.tar.gz"
+  "strip_prefix": "rules_platform-0.1.0",
+  "url": "https://github.com/bazelbuild/rules_platform/releases/download/0.1.0/rules_platform-0.1.0.tar.gz"
```

Comment:
Is this the link to the artifact created and uploaded in distro/README.md step 5?

Reply:
Yes it is.  I think once this PR is submitted. Uploading to BCR would happen and we need to correct link to the artifact. Last time there was a complaint when I did not upload the artifact beforehand.